lib/cdc/util/parse.go

```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/artie-labs/transfer/lib/debezium"
 )
 
-// ParseField returns a `parsedValue` as type interface{}
-func parseField(field debezium.Field, value interface{}) interface{} {
+// ParseField returns a `parsedValue` as type any
+func parseField(field debezium.Field, value any) interface{} {
```

Comment:
Do you want to change the one at the end too?

Reply:
sure